doc/user/content/ingest-data/network-security/static-ips.md

```diff
@@ -31,7 +31,24 @@ connections from your region's egress IP addresses. You must allow connections
 from all egress IP addresses associated with your region. Connections may
 originate from any one of the egress IP addresses.
 
-To find the egress IP addresses associated with your region, you can query the
+
+Each region of materialize uses a static set of CIDRs for egress IPs.
+
+#### AWS Static Ips
```

Comment:
nit: `IPs`

Reply:
or call this `AWS Egress IPs`? might be more obvious than static, which doesn't necessarily mean egress